crates/signature-mpc/src/twopc_mpc_protocols/mod.rs

```diff
@@ -705,3 +705,53 @@ pub fn recovery_id(
         ),
     }
 }
+
+/// Verifies that all the signatures are valid. Return true if all are valid, false otherwise.
+pub fn verify_signatures(
+    messages: Vec<Vec<u8>>,
+    hash: &Hash,
+    public_key: PublicKeyValue,
+    signatures: Vec<Vec<u8>>,
+) -> bool {
+    messages
+        .iter()
+        .zip(signatures.iter())
+        .all(|(message, signature)| {
+            verify_single_signature(message, signature, public_key, hash).is_ok()
+        })
+}
+
+pub fn verify_single_signature(
+    message: &Vec<u8>,
+    signature: &Vec<u8>,
```

Comment:
```suggestion
    message: &[u8],
    signature: &[u8],
```
see: https://rust-lang.github.io/rust-clippy/master/index.html#/ptr_arg